Apps/Playground/Scripts/config.json

```diff
@@ -237,6 +237,42 @@
             "playgroundId": "#BCYE7J#31",
             "referenceImage": "multiCamerasOutputRenderTarget.png"
         },
+        {
+            "title": "Multi camera rendering with scissors",
+            "playgroundId": "#1LK70I#35",
+            "referenceImage": "multiCameraRenderingWithScissors.png"
+        },
+        {
+            "title": "Scissor test",
+            "renderCount": 10,
+            "playgroundId": "#W7E7CF#29",
+            "referenceImage": "scissor-test.png",
+            "comments": "Reset playgroundId to #W7E7CF#12 when d3d12 mip-map issue is fixed"
```

Comment:
Is there an issue filed for this?

Reply:
There is now :)

https://github.com/BabylonJS/BabylonNative/issues/1265